cmd/gitserver/server/cleanup.go

```diff
@@ -69,10 +69,10 @@ var sgMaintenanceScript string
 
 const (
 	// repoTTL is how often we should re-clone a repository.
-	repoTTL = time.Hour * 24 * 45
+	repoTTL = time.Hour * 24 * 45 // 45d
 	// repoTTLGC is how often we should re-clone a repository once it is
 	// reporting git gc issues.
-	repoTTLGC = time.Hour * 24 * 2
+	repoTTLGC = time.Hour * 24 * 2 // 2d
```

Comment:
If we use Sasha's suggestion above, this can become:
```suggestion
	repoTTLGC = day * 2
```